### PR TITLE
Automated and manual unlisted pass/fail daily counts (bug 1183396)

### PR DESCRIPTION
**Pull request by mstriemer**
https://bugzilla.mozilla.org/show_bug.cgi?id=1183396

### CODE REVIEW

**Comment by mstriemer** [comment]
You can see some data on https://amo-review-reports.herokuapp.com/. I currently need to manually run a script to update that and push to heroku.

**Comment by magopian** [comment]
It's missing a lot of docstring, appart from that, lgtm

File: scripts/fetch_validation_data.py [hunk 1]
```diff
@@ -39,33 +40,82 @@
...
+
+    query = """
+        SELECT a.guid, v.version, la.action, la.created
+        FROM versions v
+        JOIN addons a on a.id=v.addon_id
+        JOIN log_activity_version lav ON lav.version_id=v.id
+        JOIN log_activity la ON la.id=lav.activity_log_id
+        WHERE a.guid IS NOT NULL AND a.is_listed=0 AND
+              la.action IN (42, 43) -- (PRELIMINARY_VERSION, REJECT_VERSION)
+        ;"""
```
Comment (wagnerand) [inline scripts/fetch_validation_data.py]: How does this exclude auto-approvals?

Comment (mstriemer) [inline scripts/fetch_validation_data.py]: I figured they wouldn't have a review entry, looks like they do. Good catch, I'll update it.

Comment (mstriemer) [inline scripts/fetch_validation_data.py]: Updated.

File: scripts/fetch_validation_data.py [hunk 2]
```diff
@@ -39,33 +40,86 @@
...
+        FROM versions v
+        JOIN files f on f.version_id=v.id
+        JOIN file_validation fv ON fv.file_id=f.id
+        JOIN addons a on a.id=v.addon_id
+        JOIN log_activity_version lav ON lav.version_id=v.id
+        JOIN log_activity la ON la.id=lav.activity_log_id
+        WHERE a.guid IS NOT NULL
+           AND a.is_listed=0
+           AND la.action IN (42, 43) -- (PRELIMINARY_VERSION, REJECT_VERSION)
+           AND fv.passed_auto_validation=0
```
Comment (wagnerand) [inline scripts/fetch_validation_data.py]: Very nice solution, I didn't know that column existed!

**Comment by mstriemer** [comment]
I updated the data for the graphs and now the manual reviews graph is entirely different. Does that seem accurate? https://amo-review-reports.herokuapp.com/

**Comment by mstriemer** [comment]
Looks like mid-August was about 5 reviews a day and early-September is about 30 a day.

**Pull request merged**